.github/workflows/main.yml

```diff
@@ -110,15 +113,21 @@ jobs:
     environment: PyPi-deploy
     steps:
       - uses: actions/checkout@v4
+        with:
+          submodules: recursive
 
       - uses: actions/setup-python@v5
         with:
           python-version: 3.12
+
+      # IMPORTANT: always build sdist, and then the wheel from
+      # the sdist (like it is currently done here).
```

Comment:
Could you please add the reason too?

Reply:
Sure! Done in f9b51725c0bda81a1b79668371e83c5adf89fe76.